.rubocop_todo.yml

```diff
@@ -61,15 +61,15 @@ Layout/MultilineOperationIndentation:
 Layout/SpaceInsideArrayLiteralBrackets:
   Enabled: false
 
-# Offense count: 617
+# Offense count: 613
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, EnforcedStyleForEmptyBraces, SpaceBeforeBlockParameters.
 # SupportedStyles: space, no_space
 # SupportedStylesForEmptyBraces: space, no_space
 Layout/SpaceInsideBlockBraces:
   Enabled: false
 
-# Offense count: 3680
+# Offense count: 3674
```

Comment:
This looks like a cop we either should permanently disable or we unknowingly use the opposite style to the default